src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25939.cs

```diff
@@ -0,0 +1,25 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue25939 : _IssuesUITest
+	{
+		public Issue25939(TestDevice testDevice) : base(testDevice)
+		{
+		}
+
+		public override string Issue => "Unable to Remove Thumb Image in Slider Control ";
+
+		[Test]
+		[Category(UITestCategories.Slider)]
+		public void SliderShouldChangeThumbImageAndResetIt()
+		{
+			App.WaitForElement("ChangeThumbImageSourceButton");
+			App.Click("ChangeThumbImageSourceButton");
+			App.Click("ResetThumbImageSourceButton");
+			VerifyScreenshot();
```

Comment:
Snapshots available in the latest build:
![image](https://github.com/user-attachments/assets/51f53262-c90b-43c2-a7b7-cee42025ef1f)

Could you commit the images?

Reply:
Sure!